.travis.yml

```diff
@@ -3,7 +3,7 @@ language: julia
 os:
   - linux
 julia:
-  - release
```

Comment:
`- 0.6` should be added here in place of `release`, yeah?

Reply:
Probably